databases/week2/homework_part1.sql

```diff
@@ -0,0 +1,48 @@
+-- Add a task with these attributes: title, description, created, updated, due_date, status_id, user_id
+INSERT INTO 
+    task(id, title, description, created, updated, due_date, status_id, user_id)
+VALUES 
+    (38, "Fly to the Mars", "To take a look", "2022-12-18 13:17:17",  "2022-12-18 13:17:17", "2025-12-18 13:17:17", 1, 11);
+
+-- Change the title of a task
+
+UPDATE
+    task
+SET
+    title = "Fly back home"
+WHERE
+    title like "%Mars%";
+
+-- Change a task due date
+UPDATE
+    task
+SET
+    due_date = "2024-12-18 13:17:17"
+WHERE
+    due_date > "2024-12-18";
```

Comment:
since your condition is not specific, it can affect many records. If this wasn't the intention I would rather make a very specific condition for the 1 task you would like to update.

Reply:
Thanks for advice. I was just trying to use a different options to specify which entity a should modify.

Reply:
Check, totally fine, just a fyi. :+1: